relay-server/src/services/processor.rs

```diff
@@ -504,6 +504,31 @@ impl<'a> ProcessEnvelopeState<'a> {
         self.remove_event();
         self.managed_envelope.reject_event(outcome);
     }
+
+    /// Determines whether profiles that would otherwise be dropped by dynamic sampling should be kept.
+    fn forward_unsampled_profiles(&self, global_config: &GlobalConfig) -> bool {
```

Comment:
I don't think this should be a method on the state. A free standing function in the dynamic sampling module seems more appropriate.

Reply:
Done!